app/controllers/api/quizzes_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::QuizzesController < ApplicationController
+
+  def index
+    render json: Quiz.where(active: true).map(&:serialize)
```

Comment:
Lets discuss

Reply:
This should have been - 
render json: Quiz.active
which should have automatically picked `QuizSerializer`